### ISSUE TITLE
Business Ethics - Table reading order 

### ISSUE THREAD

**Issue by mwest-WWprod** [openstax/cnx-recipes#648]
Final QA - Accessibility
GLOBAL

While verifying table reading order, we've confirmed that the order is counter intuitive (reading in a 'zig zag' pattern, instead of left to right).  The caption does read correctly.  Recording has been uploaded to Box for reference:

https://rice.box.com/s/xdl71hbbjz96gzgtk70ugedkavqc9m4t

**Comment by openstaxalina** [comment]
Link to content on production: https://cnx.org/contents/kUrGbuHs@3.2:TVY4B3Z7@4/6-3-An-Organized-Workforce

**Comment by Stackblocks** [comment]
I was unable to reproduce this on our accessibility laptop. Both reading straight through and stepping through the table manually read correctly. That said, it did read out the caption twice before the table and once afterwards.

**Comment by Stackblocks** [comment]
# Auto-Proceeding Read Video
https://rice.box.com/s/lxy3so1duw5ku2xcd6myzgggunwcros4

# Manually-Proceeding Read Video
https://rice.box.com/s/j5jbe35dcq9pij4mhxk393q82lm3926o

<blockquote><img src="https://cdn01.boxcdn.net/_assets/img/favicons/favicon-XEzygu.ico" width="48" align="right"><div><strong><a href="https://rice.box.com/s/lxy3so1duw5ku2xcd6myzgggunwcros4">Box</a></strong></div></blockquote>
<blockquote><img src="https://cdn01.boxcdn.net/_assets/img/favicons/favicon-XEzygu.ico" width="48" align="right"><div><strong><a href="https://rice.box.com/s/j5jbe35dcq9pij4mhxk393q82lm3926o">Box</a></strong></div></blockquote>
